### ISSUE TITLE
Missing dependencies in yolox

### ISSUE THREAD

**Issue by cjolivier01** [MCG-NJU/MixSort#1]
Among other missing things, missing MOTDataset for:
```python
  from yolox.data import MOTDataset, ValTransform
```
Do you know which commit of yolox (and which repo) you branched yolox from?

**Comment by ret-1** [comment]
Hi, 

Thanks for bringing this issue to my attention. Our code is actually built starting from this commit: https://github.com/ifzhang/ByteTrack/commit/8d47ea56f4523276eb46386625471afcb11f09d4

As for the import error you're experiencing, it's strange because I didn't encounter such issue during my experiments and tests. This may be due to an incorrect or incomplete setup of the YOLOX environment. I would suggest you to reinstall YOLOX following the instructions in our README.

Please let me know if you continue to experience this problem after reinstalling.

**Comment by tteresi7** [comment]
@ret-1 to elaborate, yolox.data does not contain a folder named datasets.  `MOTDataset` is supposed to be contained within a folder named `datasets` as per the commit you pointed towards.  However, this isn't an installation issue, it's an issue with the current master branches format.  I believe you need to add in that datasets folder from the commit you pointed to and it should work.

```
from .data_augment import TrainTransform, ValTransform
from .data_prefetcher import DataPrefetcher
from .dataloading import DataLoader, get_yolox_datadir
from .datasets import *
from .samplers import InfiniteSampler, YoloBatchSampler
```

**Comment by tteresi7** [comment]
@ret-1 I tried it and appears to successfully find `MOTDataset` now.  However, there appears to be a variable added by you named `return_orig_img`, which is not contained in the ByteTrack commit.  So it seems that in order for this to work you'll have to commit your current versions of the files within `datasets` otherwise I think we can assume `return_orig_img` to `False`, and delete it.

**Comment by ret-1** [comment]
Hi @cjolivier01 and @tteresi7,

Thank you for pointing out the issue with the missing `datasets` folder in `yolox.data`. The root cause of the problem was indeed an oversight in our `.gitignore` file where we unintentionally ignored the `datasets` folder. This has now been addressed in [Pull Request #4](https://github.com/MCG-NJU/MixSort/pull/4).

We appreciate your feedback and patience as we worked to resolve this. Please let us know if you encounter any other issues or have further feedback.

**Comment by tteresi7** [comment]
@ret-1 thanks for the update, I'll try out both this and #5 in a couple days!  Also, I didn't mention it before but thanks so much for contributing MixSort.
